### DISCUSSION TITLE
[Issue] Connection errored out on Img2Img-- intel Arc with Ipex (windows)

### DISCUSSION THREAD

**Discussion by Manstraub** [vladmandic/sdnext#2821] [Q&A] [answered]
Like the title says, if I try to do any kind of image to image generation (text to image has no problems), I just get "connection errored out", no other error messages in terminal, and have to relaunch. Using an intel arc A770 with Ipex on windows. Anyone have this issue or know a potential solution?

**Comment by Disty0** [comment, marked as answer]
Known upstream IPEX issue:
https://github.com/vladmandic/automatic/issues/2736
https://github.com/intel/intel-extension-for-pytorch/issues/505
